python/ray/data/tests/preprocessors/test_encoder.py

```diff
@@ -486,13 +569,31 @@ def test_label_encoder():
     expected_df = pd.DataFrame.from_dict(
         {"A": processed_col_a, "B": processed_col_b, "C": processed_col_c}
     )
-    assert out_df.equals(expected_df)
+    pd.testing.assert_frame_equal(out_df, expected_df)
+
+    # append mode
+    append_encoder = LabelEncoder("A", output_column="A_encoded")
+    append_encoder.fit(ds)
+    append_transformed = append_encoder.transform(ds)
+    out_df = append_transformed.to_pandas()
+
+    expected_df = pd.DataFrame.from_dict(
+        {"A": col_a, "B": col_b, "C": col_c, "A_encoded": processed_col_a}
+    )
+    pd.testing.assert_frame_equal(out_df, expected_df)
```

Comment:
for dataframe checking you should sort the dataframes ahead of time right before checking since the orders are not guaranteed

Reply:
good point, I'll add the sort

Reply:
`assert_frame_equal ` provides a `check_like` which ignores the column order but still check for row order, I'll use that